src/components/RulesModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+      <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+          <div class="modal-header">  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+            <h3 class="modal-title">Programming Wars Rules</h3>
+          </div>
+          <div class="modal-body" style="text-align: left">
+            <h4>Introduction</h4>
+            <p>The basic elements of a program are simple to understand, but can be used to create complex
+              computer programs. In this fun, fast-paced game you try to complete a simple program before
+              the opponent player or team.</p>
+
+            <h4>Components</h4>
+            <p>There are five types of cards:</p>
+            <ul>
+              <li><h5>Instruction</h5>
+              <p>Instruction cards provide the instructions used to create the
+                program. There are 24 Instruction cards divided into three types: Instruction-1,
```

Comment:
I don't know if we should be saying how many cards there are, as this may change in the future or depending on the number of players (i.e. more players means using more cards). Is knowing the number of cards important? Just state the different types. Once we have established the graphics it could be useful to show thumbnails of them in this document.